application/app_config.py

```diff
@@ -1,5 +1,5 @@
 # app_config.py
-from ConfigParser import SafeConfigParser
+from ConfigParser import SafeConfigParser, NoSectionError, NoOptionError
```

Comment:
app_config uses theres Errors in Try/Except statements, but were just not imported (this was discovered using pyflakes linter).  Currently, none of our working code actually uses the methods like `check_project_cfg_option`.